src/utils.js

```diff
@@ -8,39 +10,77 @@ const getModels = (request) => {
   const noRequestModels = !request.models;
   if (noGetDb && noRequestModels) {
     return notImplemented('`request.getDb` or `request.models` are not defined.'
-                   + 'Be sure to load hapi-sequelize before hapi-sequelize-crud.');
+      + 'Be sure to load hapi-sequelize before hapi-sequelize-crud.');
   }
 
   const { models } = noGetDb ? request : request.getDb();
 
   return models;
 };
 
-export const parseInclude = request => {
+export const parseInclude = async(request) => {
```

Comment:
hmmm… so two questions:

1. Why does it need to be async? Would a simple recurse do the job?
2. `getModelInstance` looks really dense, maybe it should be it's own method instead of a closure? It would allow us to unit test it, which might be a good idea.